### ISSUE TITLE
Not getting street map on final project template

### ISSUE THREAD

**Issue by tim-ness** [DS4PS/cpp-526-fall-2020#27]
I watching the tutorial for the final dashboard project. When Prof. Crawford opens the template it already has the full street map with street names and the data points for accidents on top of that map. When I run the template I get the data points on a blank grey page, with no street map underneath. I haven't changed anything with the template yet.
![dashboard template map](https://user-images.githubusercontent.com/69982382/95033165-10a1a700-0672-11eb-8eaf-6cf03c552926.jpg)

**Comment by jamisoncrawford** [comment]
@tim-ness that's unexpected and unusual.  There are two templates available - one for the premade dashboard and another for the "isolated" dashboard tab (found in the course schedule).  Does this happen when you "Run App" for both of those files?

**Comment by tim-ness** [comment]
@jamisoncrawford yes
![isolated tab](https://user-images.githubusercontent.com/69982382/95107631-02de3700-06ef-11eb-9c2c-32c836273655.jpg)

**Comment by jamisoncrawford** [comment]
@tim-ness can you provide the messages printed in the "R Markdown" tab?  This seems like it would be an issue with package `ggmap` or `leaflet`.  It's able to plot the correct x- and y- (longitude and latitude) coordinates but not the actual base map.

What's more - could you send me one of the files you're working on and I can try to reproduce the error?

@lecy any thoughts on this?

**Comment by tim-ness** [comment]
This is for the isolated tab



  |............                                                          |  17%
  ordinary text without R code

  |.......................                                               |  33%


processing file: 2020-02-25_isolated-tab.rmd
label: unnamed-chunk-1 (with options) 
List of 1
 $ include: logi FALSE


Attaching package: 'DT'

The following objects are masked from 'package:shiny':

    dataTableOutput, renderDataTable

Loading required package: ggplot2
Google's Terms of Service: https://cloud.google.com/maps-platform/terms/.
Please cite ggmap if you use it! See citation("ggmap") for details.

Attaching package: 'pander'

The following object is masked from 'package:shiny':

    p

Loading required package: viridisLite

Attaching package: 'rsconnect'

The following object is masked from 'package:shiny':

    serverInfo

-- Attaching packages --------------------------------------------------- tidyverse 1.3.0 --
v tibble  3.0.3     v dplyr   1.0.2
v tidyr   1.1.2     v stringr 1.4.0
v readr   1.3.1     v forcats 0.5.0
v purrr   0.3.4     
-- Conflicts ------------------------------------------------------ tidyverse_conflicts() --
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()
  |...................................                                   |  50%
  ordinary text without R code

  |...............................................                       |  67%
label: unnamed-chunk-2
  |..........................................................            |  83%
  ordinary text without R code

  |......................................................................| 100%
label: unnamed-chunk-3

output file: C:/Users/tpness/AppData/Local/Temp/Rtmpakcbcm/2020-02-25_isolated-tab.knit.md

"C:/Program Files/RStudio/bin/pandoc/pandoc" +RTS -K512m -RTS "C:/Users/tpness/AppData/Local/Temp/Rtmpakcbcm/2020-02-25_isolated-tab.utf8.md" --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output pandoc404c420274ac.html --email-obfuscation none --standalone --section-divs --template "C:\Users\tpness\Documents\R\R-4.0.2\library\flexdashboard\rmarkdown\templates\flex_dashboard\resources\default.html" --include-in-header "C:\Users\tpness\AppData\Local\Temp\Rtmpakcbcm\rmarkdown-str404c4cfb1792.html" --id-prefix section- --variable "theme:spacelab" --mathjax --variable "mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" --include-in-header "C:\Users\tpness\AppData\Local\Temp\Rtmpakcbcm\file404c197c1c52html" --include-after-body "C:\Users\tpness\AppData\Local\Temp\Rtmpakcbcm\file404c42454c55.html" --highlight-style pygments --include-before-body "C:\Users\tpness\AppData\Local\Temp\Rtmpakcbcm\file404c661d5e9a.html" --include-after-body "C:\Users\tpness\AppData\Local\Temp\Rtmpakcbcm\file404c1aad64eb.html" 
Warning in readLines(source_file) :
  incomplete final line found on 'C:/Users/tpness/Downloads/2020-02-25_isolated-tab.rmd'

Output created: C:/Users/tpness/AppData/Local/Temp/Rtmpakcbcm/file404c2a2653ad.html

**Comment by lecy** [comment]
I have not tried running the code, but if it's a map tile problem it might be one of two things: 

(1) Google has added some access restrictions for map tiles available through ggmap() and I believe ggplot(). It is still free (up to something like 40,000 queries a month), but you might have to register as a Google use and get a key.
(2) If you are pulling from Open Street Maps their severs sometimes get overloaded and might not work for an hour, then work fine the next day. Try it again? 

I can't think of anything related to your OS. Are you using a PC or Mac? (if the code works for one person and not another it is often differences in the OS). 

And last thought would be to update packages. With a big release of major code changes like R 4.0 there are always bugs to fix in the first few months and updates the packages need to do to adapt code. So you end up with a few core R updates (we are up to R 4.0.2 now?) and lots of package changes right after a major update. 

Otherwise I have not seen the new template, so I would need to open it up and do some de-bugging.

**Comment by tim-ness** [comment]
I am using a Dell PC. I did update the packages and try reloading the templates at different intervals, and I am having the same result now. I may try looking into the Google key later but it looks like I should be able to complete the assigned tasks for the project regardless of the map situation, so I'm going to focus on that for the moment. Thank you for the quick replies!
